Formula/libogg.rb

```diff
@@ -4,6 +4,8 @@ class Libogg < Formula
   url "https://ftp.osuosl.org/pub/xiph/releases/ogg/libogg-1.3.5.tar.gz"
   sha256 "0eb4b4b9420a0f51db142ba3f9c64b333f826532dc0f48c6410ae51f4799b664"
   license "BSD-3-Clause"
+  revision 1
```

Comment:
I don't think this needs a revision, if people really need this change they can reinstall instead.